python/dask_cudf/dask_cudf/io/tests/test_json.py

```diff
@@ -14,8 +14,9 @@ def test_read_json(tmp_path):
     df1 = dask.datasets.timeseries(
         dtypes={"x": int, "y": int}, freq="120s"
     ).reset_index(drop=True)
-    df1.to_json(tmp_path / "data-*.json")
-    df2 = dask_cudf.read_json(tmp_path / "data-*.json")
+    json_path = str(tmp_path / "data-*.json")
```

Comment:
I thought there was a CSV test as well?

Reply:
ah missed it, two more tests need to be fixed.

Reply:
@kkraus14 added them as well